From a8dc7fa536668842ade0aa44eb5101c88dec50c1 Mon Sep 17 00:00:00 2001
From: Manikanta Mylavarapu <quic_mmanikan@quicinc.com>
Date: Mon, 25 Sep 2023 19:09:27 +0530
Subject: [PATCH] scripts: Makefile: Add __QCOM_NON_SECURE_PIL__ to dtc flags

This change adds config to enable non-secure specific
DT properties.

Change-Id: Ic12b5cb0bc43ac366167248abe4f36df5a5b3836
Signed-off-by: Manikanta Mylavarapu <quic_mmanikan@quicinc.com>
---
 scripts/Makefile.lib | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/scripts/Makefile.lib b/scripts/Makefile.lib
index ff4cded170b4..a2047d6c3d88 100644
--- a/scripts/Makefile.lib
+++ b/scripts/Makefile.lib
@@ -261,6 +261,10 @@ ifeq ($(CONFIG_CPU_THERMAL),y)
 dtc_cpp_flags += -D __CPU_THERMAL__
 endif
 
+ifeq ($(CONFIG_QCOM_NON_SECURE_PIL),y)
+dtc_cpp_flags += -D __QCOM_NON_SECURE_PIL__
+endif
+
 ifdef CONFIG_OBJTOOL
 
 objtool := $(objtree)/tools/objtool/objtool
-- 
2.34.1

